internal/cli/main.go

```diff
@@ -189,16 +246,17 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "down":
-		if migraterErr != nil {
-			log.fatalErr(migraterErr)
-		}
-
-		downFlagSet := flag.NewFlagSet("down", flag.ExitOnError)
+		downFlagSet, helpPtr := newFlagSetWithHelp("down")
 		applyAll := downFlagSet.Bool("all", false, "Apply all down migrations")
 
-		args := flag.Args()[1:]
 		if err := downFlagSet.Parse(args); err != nil {
-			log.fatalErr(err)
+			log.Println(err)
```

Comment:
We should be consistent about how parse failures are handled in sub-commands. Previously, both the `down` and `drop` sub-commands would `log.fatalErr` so let's continue doing that.